app/javascript/dashboard/components/ChatList.vue

```diff
@@ -52,6 +64,14 @@
       @close="onCloseAddCustomViewsModal"
     />
 
+    <delete-custom-views
+      v-if="showDeleteCustomViewsModal"
+      :show-delete-popup.sync="showDeleteCustomViewsModal"
+      :close-delete-popup="onCloseDeleteCustomViewsModal"
```

Comment:
Make this an event `@close`

Reply:
Fixes by https://github.com/chatwoot/chatwoot/pull/3780/commits/d5e4f4609296da874efbb94dfa176554200fc980